webapp/client/src/pages/RevocationPage.js

```diff
@@ -27,6 +27,10 @@ export default function RevocationPage({ stepHeader, form, fields }) {
             label={{
               text: t("unlockCodeRevocation.idnr.labelText"),
             }}
+            details={{
+              title: t("unlockCodeRevocation.idnr.help.title"),
+              text: t("unlockCodeRevocation.idnr.help.text"),
+            }}
```

Comment:
I am really not sure if we need to set this here or if these should be part of the IDNr and could be toggled on and off, but that's maybe a discussion for another time 🤷

Reply:
Yes. And I think that's the same discussion as with the [comment](https://github.com/digitalservice4germany/steuerlotse/pull/367#discussion_r755020047) above. We have some texts that just always belong to one component. Question is if that coincidence and we leave it like this or if it's desired and we pull the messages into the component. Let's talk about it?